core/templates/packages.el.template

```diff
@@ -1,31 +1,63 @@
-;;; packages.el --- %LAYERNAME% Layer packages File for Spacemacs
+;;; packages.el --- %LAYER_NAME% layer packages file for Spacemacs.  -*- lexical-binding: t; -*-
```

Comment:
I have an opinion (but just an opinion) that lexical binding well be used for all new code, because it's easier to reason about. :)